### ISSUE TITLE
Exclude partner field from merchant account form in the dashboard

### ISSUE THREAD

**Issue by roadsideseb** [django-oscar/django-oscar-mws#1]
Creating a merchant account in the dashboard allows to select a partner although each Amazon merchant account is equivalent to a new partner. As a result, we shouldn't allow for picking a partner but create a new partner for the merchant account.

**Comment by natea** [comment]
Yeah, this confused me too. When I created a new Merchant account, and then went to the Partners page, there was a partner called "Amazon Amazon.com (US)"

**Comment by roadsideseb** [comment]
I've added a helptext in acb5ed085a0556f307f85098ed73ed7f873ccc1e that should make it a bit clearer. I've also change it so that the partner name (when auto-generated) is the same as the name of the `MerchantAccount`. Do you think that makes sense or is it clearer to remove the partner completely from the form and do a `get_or_create` on the partner? I'm a bit worried about side effects when partners already exist and not being able to change a wrong partner in the dashboard. I'd appreciate your thoughts.
